build-script/code-serving-doc-parser/src/common/msg_queue.py

```
@@ -0,0 +1,152 @@
+        self._queue_name = queue_name
+        self._queue_bind_routing_key = queue_bind_routing_key
+
+        self._connection: Union[BlockingConnection, None] = None
+        self._channel: Union[BlockingChannel, None] = None
+
+        self._logger = Logger.getLogger(__name__)
+
+        for i in range(20):
+            try:
+                self._connect()
+                self._logger.info(f'MQ Connected.')
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)